promql/functions.go

```diff
@@ -814,11 +863,11 @@ var functions = map[string]*Function{
 		Call:       funcResets,
 	},
 	"round": {
-		Name:         "round",
-		ArgTypes:     []model.ValueType{model.ValVector, model.ValScalar},
-		OptionalArgs: 1,
-		ReturnType:   model.ValVector,
-		Call:         funcRound,
+		Name:       "round",
+		ArgTypes:   []model.ValueType{model.ValVector, model.ValScalar},
+		Variadic:   true,
```

Comment:
Mh, so we just ignore them.

Not feeling too strongly but a mix doesn't sound good. For anything somewhat type-safe variadic arguments are furthest stretch of flexibility. Having the same in Prometheus would make for a clearer language and prevent any future messing around with optional args of different types.